src/amber/cli/templates/auth.cr

```diff
@@ -24,7 +25,23 @@ module Amber::CLI
       @timestamp = Time.now.to_s("%Y%m%d%H%M%S")
       @primary_key = primary_key
       @visible_fields = @fields.reject(&.hidden).map(&.name)
-      add_route
```

Comment:
Is there a good reason to get rid of the method and write it all inline here? I know you're calling a function that builds this, but maybe it should still be extracted out from initialize?